### ISSUE TITLE
[bug]:  PARSING_ERROR Unable to parse variables inside request body

### ISSUE THREAD

**Issue by yvz5** [hoppscotch/hoppscotch#3675]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When I use a variable inside the request body I get the following error: 
```
  PARSING_ERROR Unable to parse -
  {"url": "http://<<testapihost>>:8081"}
   (ENV_EXPAND_LOOP)
  REQUEST_ERROR Invalid URL
```
my env.json is
```
{
  "name": "test-variables",
  "variables": [
    {
      "key": "testapihost",
      "value": "localhost"
    }
  ]
}
```

the part where the collection has the variable:
```
"body": {
                        "contentType": "application/json",
                        "body": "{\"url\": \"http://<<testapihost>>:8081\"}"
                    },
```

### Steps to reproduce

1- create an env variable
2- create a request
3- add this env variable into the body
4- run `hopp test -e env.json collection.json`

### Environment

Production

### Version

Local

**Comment by jamesgeorge007** [comment]
Hi, please share a sample collection and environment export that can be used to reproduce the issue.

**Comment by jamesgeorge007** [comment]
Thanks for the sample exports. I reproduced the issue and have raised a [fix](https://github.com/hoppscotch/hoppscotch/pull/3687).

This happened since the [single environment export format](https://docs.hoppscotch.io/documentation/clients/cli#_1-single-environment-entry-export-format) only considered the first variable entry and the resolution step failed to find a match for other entries ending up in the `ENV_EXPAND_LOOP` state.

https://github.com/hoppscotch/hoppscotch/blob/7db7b9b0683424a594b85ac5265c4a5ff749586b/packages/hoppscotch-cli/src/options/test/env.ts#L39-L42

https://github.com/hoppscotch/hoppscotch/blob/7db7b9b0683424a594b85ac5265c4a5ff749586b/packages/hoppscotch-data/src/environment/index.ts#L34-L55

**Comment by jamesgeorge007** [comment]
This is fixed in [CLI v0.5.1](https://www.npmjs.com/package/@hoppscotch/cli/v/0.5.1).
